### PR TITLE
Add support for `string[][]` and `stream<string[], error?>` in addBatch

### PR DESCRIPTION
**Pull request by sachinira**
# Description
Enhance the capability of `addBatch` operation to support input types `string[][]` and `stream<string[], error?>` to cover CSV insert scenarios  https://ballerina.io/learn/by-example/io-csv.html?is_ref_by_example=true

Fixes: https://github.com/ballerina-platform/ballerina-extended-library/issues/183

One line release note: 
- New feature to support `string[][]` and `stream<string[], error?>` in addBatch

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Added new test cases for
- [x] insertCsvStreamFromFile
- [x] insertCsvStringArrayFromFile

**Test Configuration**:
* Ballerina Version: Slbeta 4
* Operating System: Ubuntu 20.04 LTS
* Java SDK: 11

# Checklist:

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: sfdc/modules/bulk/utils.bal [hunk 1]
```diff
@@ -24,6 +24,8 @@ import ballerina/lang.'xml as xmllib;
 import ballerina/regex;
 import ballerinax/sfdc;
 
+isolated string csvContent = "";
```
Comment (Chuhaa) [inline sfdc/modules/bulk/utils.bal:27]: Can you check other places also? Similar thing is repeated in utils.bal
```suggestion
isolated string csvContent = EMPTY_STRING;
```

File: sfdc/modules/bulk/client.bal [hunk 2]
```diff
@@ -222,6 +227,10 @@ public isolated client class Client {
         } else if (bulkJob.jobDataType == CSV) {
             if (content is io:ReadableByteChannel) {
                 payload = check convertToString(content);
+            } else if (content is stream<string[], error?>) {
+                payload = check convertStringListToString(content);
+            } else if (content is string[][]) {
+                payload = check convertStringListToString(content);
```
Comment (Chuhaa) [inline sfdc/modules/bulk/client.bal:233]:
```suggestion
            } else if (content is stream<string[], error?> || content is string[][]) {
                payload = check convertStringListToString(content);
```

File: sfdc/modules/bulk/utils.bal (hunk 1 above)
Comment (sachinira) [reply sfdc/modules/bulk/utils.bal]: Fixed in 2ea4f30e2ed08309e52e4026d558a483d059f7a6 . Thanks

File: sfdc/modules/bulk/client.bal (hunk 2 above)
Comment (sachinira) [reply sfdc/modules/bulk/client.bal]: Fixed in 2ea4f30e2ed08309e52e4026d558a483d059f7a6 . Thanks

**Review by Chuhaa — APPROVED** [review]

**Review by LakshanSS — APPROVED** [review]

**Pull request merged**